tests/pom.xml

```diff
@@ -257,13 +257,18 @@
             <configuration>
               <systemPropertyVariables>
                 <ufs>alluxio.underfs.s3.S3UnderStorageCluster</ufs>
-                <!-- Change "myAccessKey" to your access key for this test -->
+                <!-- Change "myAccessKey" to your access key for this test if using s3n -->
                 <fs.s3n.awsAccessKeyId>myAccessKey</fs.s3n.awsAccessKeyId>
-                <!-- Change "mySecretKey" to your secret key for this test -->
+                <!-- Change "mySecretKey" to your secret key for this test if using s3n -->
                 <fs.s3n.awsSecretAccessKey>mySecretKey</fs.s3n.awsSecretAccessKey>
-                <!-- Change "s3n://my-bucket/alluxio-test" to the name of your
-                     S3 bucket for this test -->
-                <s3Bucket>s3n://my-bucket/alluxio-test</s3Bucket>
+                <!-- Change "myAccessKey" to your access key for this test if using s3a -->
+                <aws.accessKeyId>myAccessKey</aws.accessKeyId>
+                <!-- Change "mySecretKey" to your secret key for this test if using s3a -->
+                <aws.secretKey>mySecretKey</aws.secretKey>
+                <!-- Change "s3a://my-bucket/alluxio-test" to the name of your S3 bucket & dir -->
+                <s3Bucket>s3a://my-bucket/alluxio-test</s3Bucket>
+                <!-- Increase the buffer time of journal writes to speed up tests -->
```

Comment:
Updated, thanks. I originally omitted because I thought swift would always be against a simulated cluster.